src/db/config.ts

```diff
@@ -0,0 +1,14 @@
+import { Sequelize } from 'sequelize';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const { DB_DATABASE, DB_USERNAME, DB_PASSWORD, DB_HOST, DB_DIALECT } = process.env;
```

Comment:
We will change the environment variables names to these names, and we will not take the dialect here
```suggestion
const { NODE_ENV, DEV_DB_URL, TEST_DB_URL, DATABASE_URL, } = process.env;
```